Code/GraphMol/resMolSupplierTest.cpp

```diff
@@ -713,6 +713,95 @@ void testCrambin() {
   delete crambin;
 }
 
+void testConjGrpPerception() {
+  BOOST_LOG(rdInfoLog) << "-----------------------\n"
+                       << "testConjGrpPerception" << std::endl;
+  RWMol *mol1 = MolBlockToMol(
```

Comment:
In modern C++ we can use raw string literals and no longer have to do these horrible multi-line strings.
Take a look at https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/FileParsers/testMolSupplier.cpp#L2375

Reply:
@greglandrum Done.